core/src/main/scala/io/finch/internal/currentTime.scala

```diff
@@ -1,12 +1,12 @@
 package io.finch.internal
 
+import java.time.{Instant, ZoneId}
 import java.time.format.DateTimeFormatter
-import java.time.Instant
-import java.time.ZoneOffset
+
 
 object currentTime {
   private[this] val formatter: DateTimeFormatter =
-    DateTimeFormatter.RFC_1123_DATE_TIME.withZone(ZoneOffset.UTC)
+    DateTimeFormatter.ofPattern("EEE, dd MMM yyyy HH:mm:ss zzz").withZone(ZoneId.of("GMT"))
```

Comment:
Although honestly I'm not even sure what part of the pattern would take the `Locale` into consideration. Maybe the `GMT` part, in which case I think both `Locale.ENGLISH` and `Locale.US` would do the same thing.